crates/wysiwyg/src/tests/test_to_plain_text.rs

```diff
@@ -0,0 +1,212 @@
+// Copyright 2022 The Matrix.org Foundation C.I.C.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use crate::{dom::to_plain_text::ToPlainText, ComposerModel};
+use widestring::Utf16String;
+
+#[test]
+fn text() {
```

Comment:
Maybe add a test for indented lists as well ? Since it is one of our latest enabled feature, and I guess if it crashes we want to be sure we recover nicely.

Reply:
[`cb7d3db`](https://github.com/matrix-org/matrix-rich-text-editor/pull/573/commits/cb7d3db0d962740a81f781a514a19045930ae160)